templates/terraform/examples/cloudbuild_trigger_build.tf.erb

```diff
@@ -0,0 +1,29 @@
+resource "google_cloudbuild_trigger" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
The test generated from this example fails:
```
------- Stdout: -------
=== RUN   TestAccCloudBuildTrigger_cloudbuildTriggerBuildExample
=== PAUSE TestAccCloudBuildTrigger_cloudbuildTriggerBuildExample
=== CONT  TestAccCloudBuildTrigger_cloudbuildTriggerBuildExample
TestAccCloudBuildTrigger_cloudbuildTriggerBuildExample: testing.go:674: Step 0 error: config is invalid: "github": one of `github,trigger_template` must be specified
--- FAIL: TestAccCloudBuildTrigger_cloudbuildTriggerBuildExample (0.46s)
FAIL
```

My guess is even when that is fixed, the test will probably still fail because some of the references in this example are to things that don't exist. https://github.com/GoogleCloudPlatform/magic-modules#testing-your-changes talks about testing changes. We can run the tests for you, but it's obviously faster if you can iterate on them yourself before sending it back :)

Reply:
Oh this is strange I thought I did run all the tests and confirm they passed before making PRs ready. Will look at this again this evening 🤔

Reply:
@danawillow okay I see it was because I was testing with the general Terraform provider, when I ran the test on the Terraform beta provider I got the same error. [I made a the correction to the test example here](https://github.com/GoogleCloudPlatform/magic-modules/pull/3627/commits/2caf210e6172fdb6af1082136d02885a863e1dc8) and now all tests are passing again 🤷 

Anything else to look at?

Reply:
Want to note I still need to fill out some of the specific sub resources mentioned in the original ticket. Just want to ensure things are looking good before I proceed and knock the rest out (including a test that includes a comprehensive build example)